gulp-resx2 won't build on node > 4

Attempting to install `gulp-resx2` on node > 4 results in a `libxmljs` build error. Tested on node v6.9.1. 

Updating `libxmljs` to `0.18` fixes the issue. 

Related to #2 

```
make: *** [Release/obj.target/xmljs/src/libxmljs.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/erc/.nvm/versions/node/v6.9.1/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:106:13)
gyp ERR! stack     at ChildProcess.emit (events.js:191:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:215:12)
gyp ERR! System Darwin 16.1.0
gyp ERR! command "/Users/erc/.nvm/versions/node/v6.9.1/bin/node" "/Users/erc/.nvm/versions/node/v6.9.1/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/erc/src/gulp-resx2/node_modules/libxmljs
gyp ERR! node -v v6.9.1
gyp ERR! node-gyp -v v3.4.0
gyp ERR! not ok
npm ERR! Darwin 16.1.0
npm ERR! argv "/Users/erc/.nvm/versions/node/v6.9.1/bin/node" "/Users/erc/.nvm/versions/node/v6.9.1/bin/npm" "install"
npm ERR! node v6.9.1
npm ERR! npm  v3.10.8
npm ERR! code ELIFECYCLE

npm ERR! libxmljs@0.15.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the libxmljs@0.15.0 install script 'node-gyp rebuild'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the libxmljs package,
npm ERR! not with npm itself.
```

--- a/package.json
+++ b/package.json
@@ -1,10 +1,10 @@
 {
   "name": "gulp-resx2",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "description": "A resx file converter plugin for gulp",
   "main": "index.js",
   "dependencies": {
-    "libxmljs": "0.15.0",
+    "libxmljs": "^0.18.0",
     "gulp-util": "^3.0.1",
     "through2": "^0.6.3"
   },

